lib/src/sql/statements/remove/param.rs

```diff
@@ -36,7 +36,10 @@ impl RemoveParamStatement {
 		// Clear the cache
 		run.clear_cache();
 		// Delete the definition
-		let key = crate::key::database::pa::new(opt.ns(), opt.db(), &self.name);
+		let ns_db = run.get_ns_db_ids(opt.ns(), opt.db()).await?;
```

Comment:
We assume we DO want to fail when you attempt to remove Param under an inexistent ns/db/tb, as opposed to how we implicitly create ns/db/tb for e.g. CREATE and INSERT statements.
Is that a right assumption? @tobiemh